components/kyma-environment-broker/internal/cls/config.go

```diff
@@ -158,3 +166,14 @@ func (r Region) validate() error {
 
 	return fmt.Errorf("unsupported region: %s (%s supported only)", r, strings.Join(supportedRegions, ","))
 }
+
+func ParseTemplate() (*template.Template, error) {
```

Comment:
This does not have much to do with CLS config. Let's separate it into another file `overrides.go`.

Reply:
done